pkgs/applications/office/spice-up/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitHub
+, gettext
+, libxml2
+, pkgconfig
+, gtk3
+, granite
+, gnome3
+, json_glib
+, cmake
+, ninja
+, libgudev
+, libevdev
+, vala
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "spice-up-${version}";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Philip-Scott";
+    repo = "Spice-up";
+    rev = version;
+    sha256 = "0cbyhi6d99blv33183j6nakzcqxz5hqy9ijykiasbmdycfd5q0fh";
+  };
+  patches = [ ./0001-Remove-environment-dependant-CMake-configuration.patch ];
+
+  XDG_DATA_DIRS = [ "${granite}/share" "${gnome3.libgee}/share" ];
+
+  # Add extra packages to the VAPI path
+  # There is no environment variable, so we must edit the CMakeLists.txt
+  postPatch = ''
```

Comment:
No longer necessary?

Reply:
Err, so when testing without removing the postPatch, it is broken.

So vala uses the [this function](https://github.com/GNOME/vala/blob/cd7f8f8ec99e5b4539c91d349a5229f7728351f4/vala/valacodecontext.vala#L562-L593) to lookup vapis in the path.  It calls `g_get_system_data_dirs ()`. That uses XDG spec:

>  $XDG_DATA_DIRS defines the preference-ordered set of base directories to search for data files in addition to the $XDG_DATA_HOME base directory. The directories in $XDG_DATA_DIRS should be **seperated with a colon ':'. **

So it was just broken because of a colon (whereas nix seems to default to space separated).